clients/cobol-lsp-vscode-extension/README.md

```diff
@@ -93,28 +134,60 @@ Copybooks downloaded from the mainframe using an older version of COBOL Language
 
 The extension includes the following copybook support features:
 
-* Semantic analysis for keywords, variables, and paragraphs across copybooks, to ensure and maintain compatibility of copybooks called in code.
-* Inbuilt protection against recursive and missing copybooks. If the copybook is missing or contains looping code, an error displays, preventing issues only being discovered when the code is executed.
-* Variables and paragraphs are defined across copybooks. This ensures consistency of code, and prevents issues in error diagnostics caused by incorrect variables or paragraphs in code.
+#### Syntax and Semantic Check
+
+Syntax and semantic analysis are enabled for keywords, variables, and paragraphs across all copybooks used in the COBOL file, to ensure and maintain compatibility of copybooks called in code.
+
+The semantic analysis feature takes into account `COPY REPLACING` statements which alter the content of copybooks when checking for errors.
+
+![Semantic check before and after a REPLACING statement](/docs/images/CPYReplacing.gif)
+
+#### Syntax Coloring
+
+Syntax coloring is automatically enabled for copybook files with the extension `.cpy`, as long as they are used in the main COBOL file.
```

Comment:
I'm not sure, but I think we should explain in more detail that the user may enable the coloring for copybooks without extensions by selecting "COBOL Copybook" language id

Reply:
Done (on both readmes)